### DISCUSSION TITLE
IconButton is no longer available in 2.19.20

### DISCUSSION THREAD

**Discussion by ovaisa** [iTwin/itwinjs-core#2620] [Q&A] [answered]
I migrated my code (from 2.19.6) to the latest react template (2.19.20) I downloaded today (Nov 1, 2021).  It seems that IconButton is no longer available in this version.  I saw this control in IssuesWidget.tsx (in frontend-showcase>issues), imported from @itwin/itwinui-react and used it in my code.  It seems that this path is no longer in @itwin folder in 2.19.20 (frontend-showcase>issues is still using it though)

Can you please guide if this control is deprecated or perhaps renamed or relocated.  I was unable to find any documentation on this control on itwin API documentation portal.

Thank you,

Ovais

**Comment by bsteinbk** [comment, marked as answer]
From what I see, `IconButton` is still available in @itwin/itwinui-react you can see it [here](https://github.com/iTwin/iTwinUI-react/blob/b863b1d8917be9b64954bce03d75f60eb5f80d7a/src/core/Buttons/IconButton/IconButton.tsx#L26).  See following [example](https://itwin.github.io/iTwinUI-react/?path=/story/buttons-iconbutton--add) usage.

**Reply by ovaisa** [reply]
Hi Bill,

I didn't realize that this was in a package that needed to be installed separately.  After installing this package (thanks for the link to the website), it is working.

Much appreciate your guidance,

Ovais

**Reply by ovaisa** [reply]
I explored this package and seems there are some very useful controls that I would like to use.  It seems there is a lot of controls that are same/similar to those in iTwin packages.  Can you please advise what is Bentley's recommendation on which controls are preferable as they seems to be very similar functionally and esthetically.

**Reply by bsteinbk** [reply]
The iTwinUI-react package is much newer and their controls have no dependency on imodeljs.  We will be deprecating most of the duplicate controls that are now in imodeljs/itwinjs-core in favor of the ones in itwinui-react.
